src/components/datepicker/FullNavigationBar.js

```diff
@@ -0,0 +1,90 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Box } from '../box';
+import { Button, ButtonGroup } from '../button';
+import {
+  IconArrowLeftMediumOutline,
+  IconArrowRightMediumOutline,
+  IconArrowLeftSmallOutline,
+  IconArrowRightSmallOutline,
+} from '@teamleader/ui-icons';
+import { Section, Heading2 } from '../..';
```

Comment:
Can you not import this from the index file please? About the circular dependencies @mikeverf talked